packages/plugin-core/src/dendronExtensionInterface.ts

```diff
@@ -70,6 +72,8 @@ export interface IDendronExtension {
   noteLookupProviderFactory: INoteLookupProviderFactory;
   schemaLookupProviderFactory: ISchemaLookupProviderFactory;
 
+  activateWatchers(): Promise<void>;
+  deactivate(): Promise<void>;
```

Comment:
Let's make it clear in the docstring that this method is meant to deactivate the entire extension, not just the watchers.  The way it's sandwiched between `activateWatchers` and `pauseWatchers` is a little bit misleading.